pyproject.toml

```diff
@@ -0,0 +1,32 @@
+[tool.black]
+target-version = ['py38', 'py39', 'py310']
+skip-string-normalization = false
+line-length = 79
```

Comment:
Why have you done this? The current repo is using `black` default of `88` which I think is sensible

Reply:
Also, if you didn't have this then you could get rid of `[tool.black]` altogether as you are just using the defaults

Reply:
Line lengths updated.
If you get rid of `[tool.black]` then the IDEs may use a different linter as default and you won't see the produced/suggested changes only till pre-commit is run. I also think keeping some defaults explicit protects of changes that may happen on their side.